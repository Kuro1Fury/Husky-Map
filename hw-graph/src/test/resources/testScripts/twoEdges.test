# Test that two edges can be added

CreateGraph A
AddNode A n1
AddEdge A n1 n1 e1
AddEdge A n1 n1 e2