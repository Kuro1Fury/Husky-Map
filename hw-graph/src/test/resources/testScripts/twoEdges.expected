# Test that two edges can be added

created graph A
added node n1 to A
added edge e1 from n1 to n1 in A
added edge e2 from n1 to n1 in A